src/extension/extension.ts

```diff
@@ -13,62 +13,341 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import { backendI18nDefaults, backendI18nDictionaries } from "@kie-tools-core/backend/dist/i18n";
 import { VsCodeBackendProxy } from "@kie-tools-core/backend/dist/vscode";
 import { EditorEnvelopeLocator, EnvelopeContentType, EnvelopeMapping } from "@kie-tools-core/editor/dist/api";
 import { I18n } from "@kie-tools-core/i18n/dist/core";
 import * as KogitoVsCode from "@kie-tools-core/vscode-extension/dist";
-import { getRedHatService, TelemetryService } from "@redhat-developer/vscode-redhat-telemetry";
+import { getRedHatService, TelemetryEvent, TelemetryService } from "@redhat-developer/vscode-redhat-telemetry";
 import * as vscode from "vscode";
-import { KAOTO_FILE_PATH_GLOB } from "./helpers";
+import { KAOTO_FILE_PATH_GLOB, isCamelPluginInstalled, verifyJBangIsInstalled } from "../helpers/helpers";
 import { VSCodeKaotoChannelApiProducer } from './../webview/VSCodeKaotoChannelApiProducer';
+import { NewCamelRouteCommand } from '../../src/commands/NewCamelRouteCommand';
+import { NewCamelKameletCommand } from '../../src/commands/NewCamelKameletCommand';
+import { NewCamelPipeCommand } from '../../src/commands/NewCamelPipeCommand';
+import { NewCamelFileCommand } from '../../src/commands/NewCamelFileCommand';
+import { NewCamelQuarkusProjectCommand } from "../../src/commands/NewCamelQuarkusProjectCommand";
+import { NewCamelSpringBootProjectCommand } from "../../src/commands/NewCamelSpringBootProjectCommand";
+import { NewCamelProjectCommand } from "../../src/commands/NewCamelProjectCommand";
+import { CamelRunJBangTask } from "../../src/tasks/CamelRunJBangTask";
+import { CamelKubernetesRunJBangTask } from "../../src/tasks/CamelKubernetesRunJBangTask";
+import { CamelAddPluginJBangTask } from "../../src/tasks/CamelAddPluginJBangTask";
+import { CamelStopJBangTask } from "../../src/tasks/CamelStopJBangTask";
+import { CamelRouteOperationJBangTask } from "../../src/tasks/CamelRouteOperationJBangTask";
+import { IntegrationsProvider, Integration } from "../views/IntegrationsProvider";
+import { HelpFeedbackProvider } from "../../src/views/HelpFeedbackProvider";
+import { OpenApiProvider } from "../../src/views/OpenApiProvider";
+import { confirmFileDeleteDialog } from '../../src/helpers/modals';
+import { ChildItem, DeploymentsProvider, ParentItem, Route } from "../views/DeploymentsProvider";
+import * as pjson from '../../package.json';
+import { DataMappingsProvider } from "../../src/views/DataMappingsProvider";
+import { TestsProvider } from "../../src/views/TestsProvider";
+import { RouteOperation } from "../helpers/CamelJBang";
+import { CamelLogJBangTask } from "../../src/tasks/CamelLogJBangTask";
+import { PortManager } from "../../src/helpers/PortManager";
```

Comment:
import can be simplified by removing `../src`

Reply:
extension.ts needs to be fully refactored

Reply:
https://github.com/djelinek/vscode-kaoto/pull/2